apps/aecontract/test/aecontract_staking_contract_SUITE.erl

```diff
@@ -102,7 +103,8 @@ groups() ->
          adjust_stake,
          withdraw,
          rewards,
-         penalties,
+         sct_penalties,
+         elct_penalties,
```

Comment:
These acronyms are not obvious to me? sct? elct?

Reply:
if they stay for staker contract and election contract, then I find it weird that you would associate penalties with contracts, they are associated to actions you perform or not perform?